filters/ratelimit/ratelimit.go

```diff
@@ -72,9 +71,10 @@ func NewRatelimit() filters.Spec {
 //    -> clusterRatelimit(200, "1m")
 //    -> "https://foo.backend.net";
 //
-// The above example behaves like the "ratelimit", i.e. per backend. To create a client limit
-// like in "localRatelimit" a third parameter is mandatory. Currently known parameters are
-// "auth" (limit by "Authorization" header) and "xfwd" (client ip from X-Forwarded-For header)
+// The above example behaves like the "ratelimit", i.e. per backend.
+// There is an optional third parameter, which is one of "auth"
+// (limit by "Authorization" header) and "xfwd" (client ip from
```

Comment:
Would it be bad if we allow to specify arbitrary header here instead of using limited number of magic strings?

Reply:
Good idea. This should be a separate pull request. Would you like to implement it?

Reply:
I can do it in this PR, but xfwd, will be separate, because it needs a separate path (think about the client ip handling)

Reply:
fixed